lib/pystray/_base.py

```diff
@@ -66,6 +66,10 @@ class Icon(object):
     #: notification.
     HAS_NOTIFICATION = True
 
+    #: Whether this particular implementation forces displaying a
+    #: large notification icon
+    FORCE_LARGE_NOTIFICATION_ICON = False
```

Comment:
These class-level constants have previously served as an indication that the backend supports a particular feature whereas this new field serves as application-wide configuration.

I understand why you have implemented it this way, but I think a more suitable approach would be to specify this with a constructor argument, but there is currently no way to pass a constructor argument to one implementation only. What do you think of a backend specific options system like [this](https://github.com/moses-palmer/pynput/blob/master/lib/pynput/keyboard/_base.py#L699)?

Reply:
When we set **NIIF_ICON_MASK** flag to **dwInfoFlags** parameter. Toast notification adds a text that is the file description of the executable. In our case it is **python.exe**.  It's not a problem for me because I will create an executable via pyinstaller and set a version file for my application. For other library user may not want to see ugly pyhon text.